src/current/v24.3/diagnostics-reporting.md

```diff
@@ -28,6 +28,10 @@ In all cases, names and other string values are scrubbed and replaced with under
 
 ## Opt out of diagnostics reporting
 
+{{site.data.alerts.callout_info}}
+Diagnostics reporting is required for [Trial or Free licenses]({% link {{ page.version.version }}/licensing-faqs.md %}#obtain-a-license). Therefore, the cluster setting `diagnostics.reporting.enabled` is ignored on clusters with Trial or Free licenses. The following opt out methods apply to clusters with [Enterprise licenses]({% link {{ page.version.version }}/licensing-faqs.md %}#obtain-a-license).
```

Comment:
it will be called 'Enterprise Free' and will be discussed on `({% link {{ page.version.version }}/licensing-faqs.md %}#types-of-licenses)`

(sorry just driving by bcs someone showed me this link bcs i filed https://github.com/cockroachdb/cockroach/issues/132688)

Reply:
(PS you can preview the new license language in #19002 )

Reply:
updated